proxysql/datadog_checks/proxysql/proxysql.py

```diff
@@ -0,0 +1,294 @@
+from collections import defaultdict
+from contextlib import closing, contextmanager
+
+import pymysql
+import pymysql.cursors
+
+from datadog_checks.base import AgentCheck
+from datadog_checks.errors import ConfigurationError
+
+GAUGE = "gauge"
+RATE = "rate"
+COUNT = "count"
+MONOTONIC = "monotonic_count"
+
+STATS_MYSQL_GLOBAL = {
+    "Active_Transactions": ("proxysql.active_transactions", GAUGE),
+    "Query_Processor_time_nsec": ("proxysql.query_processor_time_nsec", RATE),
+    "Questions": ("proxysql.questions", RATE),
+    "Slow_queries": ("proxysql.slow_queries", RATE),
+    "SQLite3_memory_bytes": ("proxysql.sqlite3_memory_bytes", GAUGE),
+    "Client_Connections_aborted": ("proxysql.client.connections_aborted", RATE),
```

Comment:
```suggestion
    "Client_Connections_aborted": ("proxysql.client.connections_aborted", GAUGE),
```

Reply:
Here we have a similar problem. When we query the proxysql metrics, this value is always increasing (it doesn't reset). But we want to see the metric as the difference between the previous and present value.

For example, here are values we query vs. values we want to see/submit in datadog
```
Values queried from proxysql:
[1, 1, 2, 2, 2, 5, 5]

Values we'd like to see in the metric:
[1, 0, 0, 0, 0, 3, 0]
```

I don't know if with the COUNT type I can achieve this